mesheryctl/internal/cli/root/workspaces/list.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package workspaces
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/eiannone/keyboard"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/system"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/server/models"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var listWorkspaceCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List all registered workspace",
+	Long:  `List all registered workspace`,
+	Example: `
+// List all registered workspace
+mesheryctl exp workspace list --orgID [orgId]
+
+// List of models with specified page number (25 workspace per page)
```

Comment:
Any specific reason to user models here?
```suggestion
// List of workspaces with specified page number (25 workspace per page)
```